test/methods.jl

```diff
@@ -0,0 +1,17 @@
+@testset "bias subtraction" begin
+    # testing non-mutating version
+    @test bias_subtraction(ones(500, 500), ones(500, 500)) == zeros(500, 500)
+
+    # testing mutating version
+    frame = ones(500, 500)
+    bias = ones(500, 500)
+    bias_subtraction!(frame, bias)
+    @test frame == zeros(500, 500)
+
+    # testing error
+    @test_throws ErrorException bias_subtraction(ones(500, 1), ones(500, 500))
+
+    # testing type stability
+    @inferred bias_subtraction(frame, bias)
+    @inferred bias_subtraction!(frame, bias)
```

Comment:
Note that `@inferred` returns the value, so you can use directly in the above tests, like
```julia
@test @inferred(bias_subtraction(ones(500, 500), ones(500, 500))) == zeros(500, 500)
```
and line 8 can use `@inferred`